### PR TITLE
Understanding The Review Column

### PR DESCRIPTION
**Pull request by asradebe**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by babalwa-mbolekwa — APPROVED** [review]

File: content/topics/tilde/understanding-the-review-column/_index.md [hunk 1]
```diff
@@ -77,4 +77,10 @@ In terms of accuracy, if you mark something as Competent and then a staff member
...
 
-By reviewing code and stating your opinion you will constantly test and upgrade your skills.
\ No newline at end of file
+By reviewing code and stating your opinion you will constantly test and upgrade your skills.
+
+## Instruction:
+- You need to demonstrate your understanding of the review column before putting this topic card into the review column. 
+- To do this, FIRST, you need to do one of the project cards, The *How Repo Project work*. Once you've merged it and moved it to the review column you can then move this topic card to the review column for a review.
+
+*REMEMBER* to be marked competent on this topic card, Instructions must be followed.
```
Comment (Sbonelo01) [inline content/topics/tilde/understanding-the-review-column/_index.md:86]: Please add a comma, or "that" after *REMEMBER*

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]
Small change

**Review by ry-oc — CHANGES_REQUESTED** [review]
you deleted content/soft-skills/learning-how-to-learn/_index.md was this intentional?

**Comment by asradebe** [comment]
> you deleted content/soft-skills/learning-how-to-learn/_index.md was this intentional?

ryan yes

File: content/topics/tilde/understanding-the-review-column/_index.md (hunk 1 above)
Comment (asradebe) [reply content/topics/tilde/understanding-the-review-column/_index.md:86]: Grammarly throws an error for 'that' will use a comma

**Pull request closed without merging**